package.json

```diff
@@ -1,9 +1,20 @@
 {
-  "name": "server",
-  "version": "0.0.0",
-  "private": true,
+  "name": "Kiddo",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/VolkRiot/Kiddo.git",
+  "author": "",
+  "license": "",
   "scripts": {
-    "start": "node ./server/bin/www"
+    "start": "node ./src/server/bin/www",
+    "react": " rm -rf | webpack -w",
```

Comment:
'rm -rf' will likely not work for Steven on Windows. Instead add rimraff as a dependency. It is an npm package which will run the appropriate version of rm -rf on both environments intelligently.